### PR TITLE
fix: Clean up json schema

### PR DESCRIPTION
**Pull request by untitaker**
We only want to document errors for now, and no internal fields.

### CODE REVIEW

**Comment by untitaker** [comment]
@fpacifici If I understand the spreadsheet correctly this should resolve all remaining problems with the error schema.

File: relay-general/src/protocol/exception.rs [hunk 1]
```diff
@@ -56,7 +56,7 @@ pub struct Exception {
     #[metastructure(skip_serialization = "empty", omit_from_schema)]
     pub raw_stacktrace: Annotated<RawStacktrace>,
 
-    /// An optional value which refers to a [thread](#Thread).
+    /// An optional value which refers to a [thread](#typedef-Thread).
```
Comment (PeloWriter) [inline relay-general/src/protocol/exception.rs]:
```suggestion
    /// An optional value that refers to a [thread](#typedef-Thread).
```

**Review by PeloWriter — APPROVED** [review]

**Review by fpacifici — APPROVED** [review]
Seems good to me. Did not see anything else to remove.

**Comment by fpacifici** [comment]
Please double check the build issue. The error seems relevant and easy to fix.

**Pull request merged**